### PR TITLE
[EN] Add RMoTI1 Article

### PR DESCRIPTION
**Pull request by oralekin**
pretty much copy paste of the 2 one article

if anyone wants to tell me how to not have that merge commit, go ahead

### CODE REVIEW

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+![][flag_US] [\[Sony\]](https://osu.ppy.sh/u/8750694) (formerly known as sonyguy699) won the Invitational, beating ![][flag_PL] [CarlosH20](https://osu.ppy.sh/u/8102219) in the finals. 
+
+## Schedule
+
+The RMoT Invitational are held over two weekends.
+
+| Event                               | Date        |
+|------------------------------------:|:------------|
+| Round of 16                         | 7 Jan 2017  |
```
Comment (MegaApplePi) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: full stops after the day and prefixed zero
`07. Jan 2017`
_(suppressing other instances)_

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+# Rapid Monthly osu! Tournament 1
+
+The Rapid Monthly osu! Tournament Invitational \#1 is a tournament organized and ran by the Rapid Tournaments team.
+It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
+
+The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational1). Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/538852).
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: `The tournament brackets can be found [on Challonge](https://challonge.com/rmotinvitational1). Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/538852).`

or

`The tournament brackets can be found [on Challonge.com](https://challonge.com/rmotinvitational1). Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/538852).`

I see two fullstops.

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+|-------------------|--------|
+| Host              | ![][flag_SI] [Redavor](https://osu.ppy.sh/u/3328606)|
+| Admins            | ![][flag_SI] [DomenBzo](https://osu.ppy.sh/u/3450825), ![][flag_SI] [kristi71111](https://osu.ppy.sh/u/3826105), ![][flag_TR] [oralekin](https://osu.ppy.sh/u/7631823)|
+| Referees          | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_US] [Obilitus](https://osu.ppy.sh/u/4857153), ![][flag_US] [Cierius](https://osu.ppy.sh/u/7376116)|
+| Mappool Selectors | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_US] [Obilitus](https://osu.ppy.sh/u/4857153), ![][flag_US] [Cierius](https://osu.ppy.sh/u/7376116)|
+| Streamers         | ![][flag_US] [Rosilius](https://osu.ppy.sh/u/5939931), ![][flag_SI] [DomenBzo](https://osu.ppy.sh/u/3450825)|
+
+## Prizes
+
+The Prizes are handed out in the form of Points. every 36 points can be redeemed for a month of osu!supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: Weird sentence.

My attempt:-

`The prizes are handed out in the form of **points**; every 36 points can be redeemed for a month of osu!supporter. Invitational winners are awarded twice the amount awarded in RMoT's.`

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+The winner of the Invitational tournament gets 96 points, 2ⁿᵈ place gets 48 points etc..
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: Sup.

Also, how do you calculate the points. Divide by two?

`The winner of the Invitational tournament gets 96 points, 2nd place gets 48 points, 3rd place gets 24 points, and the points continuously divided by two the lower the placement until the points is 0.`

Not comfortable with my last line. Might as well list out the points for placement.

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
+# Rapid Monthly osu! Tournament 1
+
+The Rapid Monthly osu! Tournament Invitational \#1 is a tournament organized and ran by the Rapid Tournaments team.
+It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: Nitpick. 4 -> four.

`It is an exclusive tournament where only players who reached semi-finals in one of the last four RMoT's get invited.`

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+* ![][flag_DE] [-Rawn](https://osu.ppy.sh/u/6600592)
+* ![][flag_NL] [Senpaii Henttai](https://osu.ppy.sh/u/7093651)
+* ![][flag_US] [EZdeath](https://osu.ppy.sh/u/7030260)
+* ![][flag_US] [\[Sony\]](https://osu.ppy.sh/u/8750694) (formerly known as sonyguy699)
+* ![][flag_US] [AziiFox](https://osu.ppy.sh/u/8133870) (formerly known as Crystalsim)
+* ![][flag_SE] [ErroTheCube](https://osu.ppy.sh/u/4425262)
+* ![][flag_PL] [CarlosH20](https://osu.ppy.sh/u/8102219)
+* ![][flag_US] [Lifemarr](https://osu.ppy.sh/u/7974462)
+
+**Notice:** ErroTheCube and Senpaii Henttai both reached semi-finals twice or more. Due to this, 3 people who have not reached semi-finals were selected to participate in the tournament, filler players.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: (I have) no idea what does the notice even meant.

```Markdown
**Notice:** ErroTheCube and Senpaii Henttai both reached semi-finals twice or more. Due to this, three players who have not reached semi-finals were selected to participate in the tournament as filler players.
```

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+The filler players are chosen according to a few criteria in the following priority:
+
+1. Having participated in and reached QF (quarter-finals) once out of the last 4 RMoT's
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: last 4 -> last four

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+1. Number of times reached QF
+1. Number of times participated in RMoT
+1. Current PP
+
+![][flag_PL] [dark_assasin16](https://osu.ppy.sh/u/6178640), ![][flag_PL] [MagBlood](https://osu.ppy.sh/u/6178640) and ![][flag_PL] [MinatoIDKun](https://osu.ppy.sh/u/6178640) gained the right to participate in the Invitational, MinatoIDKun being seeded lower than the other three.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: So, how are they seeded? The wording need to be fixed but first, need context.

?
?
MinatoIDKun

?

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+## Rules
+
+### General Rules
+
+* The tournament will be held in 1v1 double-elimination format with traditional seeding.
+* The matches will be played using Score V2 and TeamVS settings.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: ```* The matches will be played using `Score V2` and `TeamVS` settings.```

File: wiki/Tournaments/RMoT_Invitational/1/en.md [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+* A referee will be present in every match and every match will be streamed, with additional commentators when possible.
+* The tournament will be held over 2 weekends, joining the Discord server is mandatory at least for the duration of the tournament.
+* First day matches will be best of 5, second day matches best of 7, third day matches are best of 9, and fourth day matches will be best of 11. 
+
+### Match Rules
+
+1. The referee will create the room and invite or move the players in.
+1. Theplayers will roll to determine the order of picking warmups and maps.
```
Comment (RaikireHiuduo) [inline wiki/Tournaments/RMoT_Invitational/1/en.md]: `1. The players will roll to determine the order of picking warmups and maps.`

**Comment by Repflez** [comment]
Just asking, is this PR ready to merge or there's another I should look at first. It's almost 1 month old, so I'm asking just in case.

**Comment by TPGPL** [comment]
Apply changes from the reviews

**Comment by oralekin** [comment]
going to PR together with the third article

**Pull request closed without merging**